doc/configuration.md

```diff
@@ -0,0 +1,21 @@
+# Introduction
+
+This guide details Ark's configuration.
+
+# R global variables
+
+## ark.ragg
```

Comment:
We also have `positron.error_entrace`.

It should probably be renamed to `ark.error_entrace`? Likely worth documenting here soon so we don't forget about it. Both for user discoverability and for us to think about API consistency as we add new items here.